src/Microsoft.PowerShell.Commands.Management/commands/management/Clipboard.cs

```diff
@@ -76,7 +76,7 @@ public static string GetText()
             }
             else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
             {
-                tool = "xclip";
+                tool = "wl-clipboard";
```

Comment:
Won't this make this not work on non-Wayland systems?  Should we just have it look for xclip and if not found, look for wl-clipboard?

Reply:
/cc @mklement0